gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.1-bin.zip
```

Comment:
Please upgrade to Gradle 7.3.3 for Log4J vulnerability mitigation 👍 
```gradle wrapper --gradle-version 7.3.3```
https://blog.gradle.org/log4j-vulnerability
https://docs.gradle.org/7.3.3/release-notes.html